src/components/HireStreamEditPopup/HireStreamEditPopup.tsx

```diff
@@ -0,0 +1,82 @@
+import { HireStream } from '@prisma/client';
+import { Button, FormControlInput, Modal, ModalContent, ModalCross, ModalHeader } from '@taskany/bricks/harmony';
+import { useForm } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+
+import { EditHireStream, editHireStreamSchema } from '../../modules/hireStreamTypes';
+import { useEditHireStreamMutation } from '../../modules/hireStreamsHooks';
+import { FormControl } from '../FormControl/FormControl';
+import { FormActions } from '../FormActions/FormActions';
+
+import { tr } from './HireStreamEditPopup.i18n';
+import s from './HireStreamEditPopup.module.css';
+
+interface HireStreamEditPopupProps {
+    visible: boolean;
+    onClose: VoidFunction;
+    hireStream: HireStream;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const parseNullishNumber = (v: any) => (v ? parseInt(v, 10) : null);
```

Comment:
Zero as value will be converting into null value, is it correctly behavior?

Reply:
For this form - yes